public/app/features/scenes/dashboard/DashboardSceneRenderer.tsx

```diff
@@ -31,18 +29,12 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
               ))}
             </div>
           )}
-          {viewPanel ? (
-            <div className={styles.viewPanel}>
-              <viewPanel.Component model={viewPanel} />
-            </div>
-          ) : (
-            <div className={styles.body}>
-              <body.Component model={body} />
-            </div>
-          )}
+          <div className={styles.body}>
+            <bodyToRender.Component model={bodyToRender} />
+          </div>
         </div>
       </CustomScrollbar>
-      {inspectPanel && <ScenePanelInspector panel={inspectPanel} dashboard={model} />}
+      {drawer && <drawer.Component model={drawer} />}
```

Comment:
was pretty happy with this change, now the dashboard renderer does not need to care about that drawer is to be shown that can be only managed by state logic